src/components/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import Navigation from '../Navigation/Navigation';
+import Forms from '../Forms/Forms';
+import miniTopFood from '../../images/mini-top-food.png';
+import restoran from '../../images/home-restoran.png';
+import meni from '../../images/meni-home.png';
+import phone from '../../images/home-phone.png';
+import FooterForHomePage from '../Footer/FooterForHomePage';
+import { RiStarSFill } from 'react-icons/ri';
+import { AiOutlineStar } from 'react-icons/ai';
+
+const HomePage = () => {
+  return (
+    <div className='forms'>
+      <div className='main-home'>
+        <Navigation />
+        <Forms />
+      </div>
+      <div className='second-box-home'>
+        <div className='first-box'>
+          <img src={miniTopFood} alt='' className='mini-top-food' />
+          <div className='paragraf-stars'>
+            <p className='paragraf-home'>RESTORANI * Domaca Kuhinja</p>
```

Comment:
Please put these 3 mocked restaurants to array and just map them, then you will have more cleaner code